packages/react-components/src/components/Announcer.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStackStyles, Stack } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * @private
+ *  aria props for this hidden component.
+ */
+export type AnnouncerProps = {
+  announcementString: string;
+  ariaLive: 'off' | 'polite' | 'assertive' | undefined;
+};
+
+/**
+ * @private
+ * Announcer component to make Aira announcments on actions
+ */
+export const Announcer = (props: AnnouncerProps): JSX.Element => {
+  const { announcementString, ariaLive } = props;
+
+  return <Stack aria-label={announcementString} aria-live={ariaLive} styles={announcerStyles}></Stack>;
+};
+
+/**
+ * Styles to hide the announcer from view but still displaying it so that narration can happen.
```

Comment:
nit: instead of `displaying` say something like `exists in the DOM tree` to make it clear nothing is visible/being displayed to the user's screen